src/common/vscode/diagnostic/diagnosticsUtil.ts

```diff
@@ -2,18 +2,46 @@ import * as vscode from "vscode";
 import { flatten } from "../../general/arrayUtil";
 import { getRangesOfProperty } from "../editor/textDocumentUtil";
 
+/**
+ * Returns Diagnostics on a text document with given property and a message.
+ * @param propertyNames Property name to search for on given text document.
+ * @param message Message of diagnostics to be created.
+ * @param document A text document to create diagnostics on.
+ * @param severity Severity of diagnostics to be created
+ */
+function createDiagnostics(
+    propertyName: string,
+    message: string,
+    document: vscode.TextDocument,
+    severity?: vscode.DiagnosticSeverity
+): vscode.Diagnostic[];
+
```

Comment:
already in here 👇

Reply:
I just realized `string` and `string[]` difference.

Reply:
there is a small change in the first parameter there, but I made it wrong appearently, I'll fix it

Reply:
I fixed the documentation